frontend/app/enrollment/task/[taskId]/page.tsx

```diff
@@ -19,14 +20,53 @@ export default function EnrollmentTaskPage() {
 
     if (loading || !initialized) return <Loading />
 
-    if (!subTasks || !subTasks.length || !taskToQuestionnaireMap || !subTasks?.[0].id || !taskToQuestionnaireMap[subTasks[0].id]) {
-        return <></>
-    }
+    const serviceName = task?.focus?.display || "Unknown"
+    const conditionName = task?.reasonCode?.coding?.[0].display || "Unknown"
+    const taskDate = task?.meta?.lastUpdated ? new Date(task.meta.lastUpdated).toLocaleDateString("nl-NL") : "Onbekend"
 
-    return (
-        <QuestionnaireRenderer
-            questionnaire={taskToQuestionnaireMap[subTasks[0].id]}
-            inputTask={subTasks[0]}
-        />
-    )
+    switch (task?.status) {
+        case "accepted":
+            return <div className='w-[568px]'>
+                Het verzoek om {serviceName} voor {conditionName} uit te voeren, is door {organizationName(task.owner)} op {taskDate} geaccepteerd. Uitvoering is nog niet gestart.
+            </div>
+        case "completed":
+            return <div className='w-[568px]'>
+                De uitvoering van {serviceName} voor {conditionName} is door {organizationName(task.owner)} afgerond op {taskDate}.
+            </div>
+        case "cancelled":
+            return <div className='w-[568px]'>
+                Het verzoek om {serviceName} voor {conditionName} uit te voeren, is op {taskDate} geannuleerd door {organizationName(task.owner)}.
+            </div>
+        case "failed":
+            return <div className='w-[568px]'>
+                Het verzoek om {serviceName} voor {conditionName} uit te voeren, is sinds {taskDate} gemarkeerd als &quot;mislukt&quot; door {organizationName(task.owner)}.
```

Comment:
maybe we should show the status reason here? It contains the rejection reason.

Or maybe we should always show it, if it's present?